includes/polling/applications/php-fpm.inc.php

```diff
@@ -0,0 +1,37 @@
+<?php
+$name = 'php-fpm';
+$app_id = $app['app_id'];
+
+$options      = '-O qv';
+$mib          = 'NET-SNMP-EXTEND-MIB';
+$oid          = '.1.3.6.1.4.1.8072.1.3.2.3.1.2.8.112.104.112.102.112.109.115.112';
+$phpfpm = snmp_walk($device, $oid, $options, $mib);
```

Comment:
Sweet. Updated it to use a NET-SNMP-EXTEND-MIB::nsExtendOutputFull.8.112.104.112.102.112.109.115.112 .